Jenkinsfile

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env groovy
+pipeline {
+    agent {
```

Comment:
I think this can just work on the "maven" label, and use the Azure Container Instance agents which would be much faster.

Reply:
+1 for ACI